recipes/openai-whisper/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "openai-whisper" %}
+{% set version = "20250625" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/openai_whisper-{{ version }}.tar.gz
+  sha256: 37a91a3921809d9f44748ffc73c0a55c9f366c85a3ef5c2ae0cc09540432eb96
+  patches:
+    - 000-utf8.patch
+
+build:
+  entry_points:
+    - whisper=whisper.transcribe:cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools >=61.2
+  run:
+    - python >={{ python_min }}
+    - numba
+    - numpy
+    - pytorch
+    - tqdm
+    - more-itertools
+    - tiktoken
+    - ffmpeg
+  run_constrained:
+    - triton >=2.0.0  # [linux64]
+
+test:
+  requires:
+    - python {{ python_min }}
+  imports:
+    - whisper
+  commands:
+    - whisper --help
```

Can you add a `- pip check` here?

```suggestion
test:
  requires:
    - python {{ python_min }}
    - pip
  imports:
    - whisper
  commands:
    - whisper --help
    - pip check
```